djl-spring-boot-console-sample/pom.xml

```diff
@@ -9,7 +9,7 @@
     </parent>
     <groupId>ai.djl.spring.examples</groupId>
     <artifactId>djl-spring-boot-console-sample</artifactId>
-    <version>0.12-SNAPSHOT</version>
+    <version>0.14.0</version>
```

Comment:
It's recommended to use `maven release` plugin to upgrade version automatically, so you don't have to change file one by one.

Reply:
Thank you for your timely reply, I understand!